server/registry/actions_migrate_database_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package registry
+
+import (
+	"context"
+	"testing"
+
+	"github.com/apigee/registry/rpc"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/genproto/googleapis/longrunning"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/testing/protocmp"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+func TestMigrateDatabase(t *testing.T) {
+	ctx := context.Background()
+	server := defaultTestServer(t)
+
+	req := &rpc.MigrateDatabaseRequest{}
+	metadata, err := anypb.New(&rpc.MigrateDatabaseMetadata{})
+	if err != nil {
+		t.Fatalf("MigrateDatabase(%+v) test failed to build expected metadata: %s", req, err)
```

Comment:
This error message doesn't seem to match the cause of the error. We're not calling `MigrateDatabase` yet, and we haven't used the `req` variable yet either. Same for the next error.

Maybe `Setup: Failed to build response message: %s`?

Reply:
These are building the *expected* response messages, not the actual ones. I'll update these to "failed to build expected response (metadata|message)".